sip/sip-001-burn-election.md

```diff
@@ -918,20 +895,20 @@ attacks.  This section will be updated as our understanding evolves.
 # Fork Selection
 
 Fork selection in the Stacks blockchain requires a metric to determine which
-chain, between two candidates, is the "best" chain.  Using proof-of-burn as the
+chain, between two candidates, is the "best" chain.  Using tunable proofs as the
```

Comment:
This paragraph is confusing to me, if fork selection is done by using the _longest valid_ fork (i.e., the fork with the most number of winning selection blocks), that's not necessarily the fork with the most cumulative proof scores.

Reply:
This line is the confusing one:

```
security method for the blockchain implies a direct metric:  the total sum of
the proof scores in the election blocks for a candidate chain.
```

I think if we're trying to say, "how secure is my transaction currently?", the answer is something like the blockchain depth of that transaction, mixed with the expected amount of work to win that many consecutive blocks. But that seems out of place here.

Reply:
This is an artifact from when we were explaining why we didn't use cumulative burn.  Will straighten out.